ray-on-gke/tpu/kuberay-tpu-webhook/main.go

```diff
@@ -134,18 +131,18 @@ func extractRayCluster(admissionReview *admissionv1.AdmissionReview) (*ray.RayCl
 	return &rayCluster, nil
 }
 
-func genDNSHostnames(workerGroupSpec ray.WorkerGroupSpec, clusterName string, namespace string, replicaIndex int) (string, error) {
-	numHosts := workerGroupSpec.NumOfHosts
-	if numHosts == 0 {
-		return "", errors.New("workerGroupSpec NumOfHosts not set")
+func genDNSHostnames(numOfHosts int32, groupName string, clusterName string, namespace string, replicaIndex int) (string, error) {
+	if numOfHosts == 0 {
+		err := errors.New("workerGroupSpec NumOfHosts not set")
+		klog.ErrorS(err, "genDNSHostnames", "RayCluster", namespace+"/"+clusterName, "NumOfHosts", numOfHosts)
```

Comment:
just return an error instead

Reply:
Done in [27ddd5d](https://github.com/GoogleCloudPlatform/ai-on-gke/pull/740/commits/27ddd5db325ab7a3856f5b8223e85d9fcb16295f).